src/FlaUILibrary/keywords/tree.py

```diff
@@ -0,0 +1,307 @@
+from FlaUILibrary.robotframework import keyword
+from FlaUILibrary.flaui.util import InterfaceType
+from FlaUILibrary.flaui.module import (Element, Tree)
+
+
+class TreeKeywords:
+    """
+    Interface implementation from robotframework usage for tree keywords.
+    """
+
+    def __init__(self, module):
+        """Constructor for tree keywords.
+
+        ``module`` Automation framework module like UIA3 to handle element interaction.
+        """
+        self._module = module
+
+    @keyword
+    def get_root_treeitems_count(self, identifier, msg=None):
+        """Return count of items in the first level of the tree.
+
+        XPath syntax is explained in `XPath locator`.
+
+        If element could not be found by xpath an error message will be thrown.
+
+        Arguments:
+        | Argument   | Type   | Description                   |
+        | identifier | string | XPath identifier from element |
+        | msg        | string | Custom error message          |
+
+        Examples:
+        | ${COUNT}  Get Root TreeItems Count  <XPATH>  |
```

Comment:
Example should include how to compare these. It's a list which is not documentated